docs/handcraft.rst

```diff
@@ -897,15 +912,15 @@ The Fulfilled Transaction
 
     In [0]: ed25519.sign(message.encode(), sk)
 
-    In [0]: fulfillment = ed25519.serialize_uri()
+    In [0]: fulfillment_uri = ed25519.serialize_uri()
```

Comment:
As a side note: the reason for which the variable was named `fulfillment` and not `fulfillment_uri` is because the key for which it holds the value is named `fulfillment` and not `fulfillment_uri`.